conf/modules/ngscheckmate.config

```diff
@@ -0,0 +1,12 @@
+process {
+    withName: ".*BAM_NGSCHECKMATE:BCFTOOLS_MPILEUP" {
+        ext.when   = { params.tools && params.tools.split(',').contains('sampleqc') }
```

Comment:
why `sampleqc` and not `ngscheckmate`?

Reply:
I was thinking that the other tools that have been discussed (sexdeterrmine, somalier) would go into the same subworkflow. So I thought they might be all controlled together, but if they should be more granular that is fine too.

Reply:
I'd rather be granular, as we already have some qc tools

Reply:
agreed. maybe someone wants to run checkmate but not somalier etc.

Reply:
To be fair, for most our QC tools, they enabled by default and disabled _à la demande_ with --skip_tools, should we do that there?